setup.py

```diff
@@ -1,16 +0,0 @@
-#!/usr/bin/env python
-
-import setuptools
-
-# In python < 2.7.4, a lazy loading of package `pbr` will break
-# setuptools if some other modules registered functions in `atexit`.
-# solution from: http://bugs.python.org/issue15881#msg170215
-try:
-    import multiprocessing  # noqa
-except ImportError:
-    pass
-
-setuptools.setup(
-    setup_requires=['pbr'],
```

Comment:
Does building via `pyproject.toml` work in the same way as with `pbr`? IIRC, pbr was putting the commit-id into some file after installation, which we use for the changelog in CI.